src/modules/data/blocks/datetime/__tests__/sagas.test.js

```diff
@@ -181,7 +181,12 @@ describe( 'Event Date time Block sagas', () => {
 
 		describe( 'onHumanReadableChange', () => {
 			test( 'On change handler when dates are null', () => {
-				const gen = sagas.onHumanReadableChange();
+				const action = {
```

Comment:
Wonder if this and the next could be cleaned up with a `beforeEach` at the top, with the action object definition and possibly the saga declaration